src/common/inline_variant.h

```diff
@@ -8,6 +8,7 @@
 #ifndef INLINE_VARIANT_H
 #define INLINE_VARIANT_H
 
+#include <boost/noncopyable.hpp>
```

Comment:
@wjwithagen could you move this `#include` after `#include <boost/mpl/range_c.hpp>`? so they are (almost) ordered alphabetically.

Reply:
@tchaikov 
I will.
My choice was to order by directory depth...